src/main/kotlin/HelloWorld.kt

```diff
@@ -2,5 +2,13 @@ fun main() {
     println("Hello world!")
     println("Hello Kotlin!")
     println("Hello Kotlin!")
-    println("Hello Kotlin!")
+
+    val tmp = "UNUSED"
+    println("Test");
+}
+
+class HelloClass {
+    fun foo() {
+        val str = "str"
+    }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/7358fcdc284f77876be97174d08432dce4b32f1f/src/main/kotlin/HelloWorld.kt#L14"></span>The file HelloWorld.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>